CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+## We develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)).
+
+1. Clone the repo and create your branch from `main`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+> When raising a pull request, ensure that you apply the appropriate label as this is used when generating changelogs!
+
+## Any contributions you make will be under the Open Government Licence
+
+In short, when you submit code changes, your submissions are understood to be under the same
+[OGLv3 License](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3) that covers the project.
+
+## Report bugs using Github's [issues](https://github.com/DEFRA/rod-licensing/issues)
```

Comment:
This still links to the rod-licencing repo

Reply:
Totally missed that, this is now fixed.